.github/workflows/build.yml

```diff
@@ -223,3 +237,26 @@ jobs:
         registry: ${{ secrets.DOCKERHUB_TOKEN && 'docker.io' || 'ghcr.io' }}
         username: ${{ secrets.DOCKERHUB_USERNAME || github.actor }}
         password: ${{ secrets.DOCKERHUB_TOKEN || github.token }}
+
+  push-release:
+    # Push image to registry after test for master.
+    if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/master' }}
```

Comment:
just for consideration, could use `github.ref_name == 'master'` as shorthand

Reply:
Yeah that'd work too, these are the same things. I'm just copying what we're already doing in other files in the repo, this section is a copy-paste. I'm going to leave it as it is to avoid refactoring things for minor benefits in this PR.